mailpoet/lib/Automation/Engine/Data/Automation.php

```diff
@@ -45,6 +45,9 @@ class Automation {
   /** @var array<string, Step> */
   private $steps;
 
+  /** @var array<string, Mixed> */
```

Comment:
Is `Mixed` a type? Should be `mixed`, I guess. But PHPStan doesn't seem to complain 🤔

Reply:
84b99e4de